x/gravity/keeper/keeper.go

```diff
@@ -32,18 +39,21 @@ type Keeper struct {
 	paramSpace paramtypes.Subspace
 
 	cdc               codec.BinaryCodec // The wire codec for binary encoding/decoding.
+	ProtoUnmarshaler  keeperutil.ProtoUnmarshaler
+	Protomarshaler    keeperutil.ProtoMarshaler
+	Unmarshaler       proto.Unmarshaler
```

Comment:
Why we need these? You can directly use codec. And I don't find where these are initialized.

Reply:
removed these too